Misc/NEWS.d/next/Library/2017-10-17-23-27-03.bpo-31806.TzphdL.rst

```diff
@@ -0,0 +1,4 @@
+Fix timeout rounding in time_sleep, lock_acquire_parse_args and
+socket_parse_timeout to round correctly negative timeouts between -1.0 and
```

Comment:
Please use the Python names, not the internal C names:

time.sleep(), threading.Lock.acquire(), socket.socket.settimeout()

Reply:
Done in d96585e